docs/contribution/release.md

```diff
@@ -0,0 +1,78 @@
+# how release works
+
+Prerequisite:
+
+* docker installed
+* How drone works
+* How [goreleaser](https://goreleaser.com/intro/) works
+* multi arch TODO, don't know yet
+
+Briefly at least.
+
+## Continuous Delivery
+
+We use drone as continuous integration and continuous delivery pipeline.
+Integration means: running tests, checking code quality.
+Delivery means: make a release when a new tag get pushed.
+
+## goreleaser
+
+GoReleaser is a popular tool to release Go applications and library.
```

Comment:
Is goreleaser expected to be installed? What installs it?

Reply:
https://goreleaser.com/install/#install-the-pre-compiled-binary

Reply:
So how does it get installed? If I run this, then I will have a failure? I think if this is used for releases only (not normal builds) then it is OK not to have it verified and installed automatically (although it would be nice, but not necessary), but we need to make it clear that: it is a prerequisite for releases; here is the link to install it; it isn't needed except for releases, which are normally handled by drone.

And, of course, we need to add it to drone.yml, but that can come later.